Presto/qubole_odbc/connection-dialog.tcd

```diff
@@ -1,15 +1,12 @@
 <connection-dialog class='qubole_odbc'>
   <connection-config>
-    <authentication-mode value='None' />
+    <authentication-mode value='PasswordOnly' />
     <authentication-options>        
-      <option name="None" default="true" value="auth-none" />
+      <option name="Password" default="true" value="api-token" />
     </authentication-options>
-    <db-name-prompt value="Catalog :" />
-    <has-databases value="true" />
+    <db-name-prompt value="@string/db_name_prompt/" />
     <has-schemas value="true" />
-    <server-prompt value="End Point;Bucket Region : " default="https://api.qubole.com;us-east-1"/>     
-    <service-prompt value="API Token : " />
-    <show-uncommitted-data-checkbox value="true" />
```

Comment:
@saugatt We use `show-uncommitted-data-checkbox` to enable streaming. I see it is removed. Do we have any specific field to achieve the same?

cc : @vijaymann